docker/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# Generate the config file from given environment variables
+
+cat << EOF > config.toml
+
+inverter_host = "$INVERTER_HOST" 
+
+[home_assistant]
+host = "$MQTT_BROKER_HOST"
+username = "$MQTT_USERNAME"
+password = "$MQTT_PASSWORD"
+port = $MQTT_PORT
+
```

Comment:
```suggestion

[simple_mqtt]
host = "$MQTT_BROKER_HOST"
username = "$MQTT_USERNAME"
password = "$MQTT_PASSWORD"
port = $MQTT_PORT
```

this would also publish the data using the simple MQTT publisher's topics.